stateproof/verify/stateproof.go

```diff
@@ -19,24 +19,25 @@ package verify
 import (
 	"errors"
 	"fmt"
-
```

Comment:
We should be able to come up with a better file name for this. 
We already have many files named `stateproof.go`, but `stateproof/verify/stateproof.go` is super special.